certbot-apache/certbot_apache/constants.py

```diff
@@ -62,9 +62,9 @@
     vhost_files="*.conf",
     logs_root="/var/log/apache2",
     version_cmd=['/usr/sbin/apache2', '-v'],
-    define_cmd=['apache2ctl', 'virtualhosts'],
-    restart_cmd=['apache2ctl', 'graceful'],
-    conftest_cmd=['apache2ctl', 'configtest'],
+    define_cmd=['/usr/sbin/apache2', '-S'],
```

Comment:
@Jarodiv I'm a OpenRC Gentoo user, using `certbot` fully automatically. It's not rocket science and works flawlessly if one keeps every virtual host in a separate file (which isn't Gentoo specific).

Also, OpenRC really is the default init system ([source 1](https://wiki.gentoo.org/wiki/Comparison_of_init_systems) [source 2](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Base#Updating_the_.40world_set)).